setup.py

```diff
@@ -95,8 +95,9 @@
     },
     project_urls={
         "Bug Tracker": "https://github.com/Qiskit/qiskit-terra/issues",
-        "Documentation": "https://qiskit.org/documentation/",
-        "Source Code": "https://github.com/Qiskit/qiskit-terra",
+        "Documentation": "https://docs.quantum.ibm.com",
+        "Source Code": "https://github.com/Qiskit/qiskit",
+        "Homepage": "https://www.ibm.com/quantum/qiskit",
```

Comment:
This appears to be a different set of links to what we merged in #11550 - was there a reason for that?